xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.cfg.vm

```diff
@@ -476,6 +476,23 @@ xwiki.authentication.ldap.fields_mapping=last_name=sn,first_name=givenName,email
 #-# The default is 0
 xwiki.authentication.ldap.update_user=1
 
+#-# [Since 8.1M2, XWikiLDAPUtils]
+#-# On every login update photo from LDAP to XWiki avatar otherwise photo will not be updated.
+#-# - 0: never
+#-# - 1: at each authentication
+#-# The default is 0
+xwiki.authentication.ldap.update_photo=0
```

Comment:
You should comment the properties and only uncomment when someone want to modify the default.